diff --git a/dependencies b/dependencies
index 48a4db2..57cf881 100644
--- a/dependencies
+++ b/dependencies
@@ -37,4 +37,3 @@ libxrandr-dev:i386
 libxi-dev:i386
 libxcursor-dev:i386
 libvulkan-dev:i386
-libvkd3d-dev:i386
diff --git a/patches-glibc/glibc1.patch b/patches-glibc/glibc1.patch
index ddcc3ef..4d9420e 100644
--- a/patches-glibc/glibc1.patch
+++ b/patches-glibc/glibc1.patch
@@ -74,12 +74,12 @@ index b70f6fa..02cd33a 100644
        /* Skip leading blanks.  */
        while (isspace (*p))
  	++p;
--    } while (*p == '\0' || *p == '#' /* Ignore empty and comment lines.  */
+-    } while (*p == '\0' || *p == '#' ||	/* Ignore empty and comment lines.  */
 +    } while (*p == '\0' || *p == '#' /* Ignore empty and comment lines.  */
  	     /* Parse the line.  If it is invalid, loop to
  		get the next line of the file to parse.  */
--	     || ! parse_line (buffer, (void *) resbuf, (void *) buffer, buflen,
--			      &errno));
+-	     ! parse_line (buffer, (void *) resbuf, (void *) buffer, buflen,
+-			   &errno));
 +	     || ! (parse_result = parse_line (p, resbuf,
 +					      (void *) buffer, buflen,
 +					      &errno)));
diff --git a/wine-lol.sh b/wine-lol.sh
index a364764..36e5da2 100755
--- a/wine-lol.sh
+++ b/wine-lol.sh
@@ -3,11 +3,11 @@ version=10.12-1
 
 PARALLEL=4
 
-glibc_src=https://ftp.gnu.org/gnu/glibc/glibc-2.30.tar.xz
+glibc_src=https://ftp.gnu.org/gnu/glibc/glibc-2.27.tar.xz
 wine_src=https://dl.winehq.org/wine/source/5.x/wine-5.9.tar.xz
 wine_staging_src=https://github.com/wine-staging/wine-staging/archive/v5.9.tar.gz
 
-glibc_md5sum=2b1dbdf27b28620752956c061d62f60c
+glibc_md5sum=898cd5656519ffbc3a03fe811dd89e82
 wine_md5sum=0a6d8a33de07a2dd25d89bd4f18626c3
 wine_staging_md5sum=b94848cbbacb010075fa4db706872aeb
 
@@ -131,8 +131,8 @@ rm sums
 if [ -d wine-5.9 ]
 	then rm -r wine-5.9
 fi
-if [ -d glibc-2.30 ]
-	then rm -r glibc-2.30
+if [ -d glibc-2.27 ]
+	then rm -r glibc-2.27
 fi
 if [ -d wine-staging-5.9 ]
 	then rm -r wine-staging-5.9
@@ -147,7 +147,7 @@ fi
 tar -xf src_glibc.tar.xz
 tar -xf src_wine.tar.xz
 tar -xf src_wine_staging.tar.gz
-ln -s glibc-2.30 glibc-src
+ln -s glibc-2.27 glibc-src
 ln -s wine-5.9 wine-src
 
 pushd wine-staging-5.9/patches
